content/v2.0/monitor-alert/send-email.md

```diff
@@ -0,0 +1,226 @@
+SENDGRID_APIKEY = secrets.get(key: "SENDGRID_APIKEY")
+
+numberOfCrits = from(bucket: "_monitoring")
+	|> range(start: -10m)
+	|> filter(fn: (r) => (r["_measurement"] == "statuses"))
+	|> filter(fn: (r) => (r["_level"] == "crit"))
+	|> count()
+
+numberOfCrits
+	|> map(fn: (r) =>
+		(if r._value > 3 then {r with _value: http.post(url: "https://api.mailjet.com/v3.1/send", headers: {"Content-type": "application/json", Authorization: "Basic <your-api-key>:<your-secret-key"}, data: bytes(v: "{
+        \"Messages\": [{
+          \"From\": {\"Email\": \”jane.doe@example.com\"},
+          \"To\": [{\"Email\": \"john.doe@example.com\"]},
+          \"Subject\": \”InfluxData critical alert\",
+          \"TextPart\": \”Example alert text\"
+          \"HTMLPart\":  `"<h3>Hello, to review critical alerts, review your <a href=\"https://www.example-dashboard.com/\">Critical Alert Dashboard</a></h3>}]}'
+              
+        }\""))} else {r with _value: 0}))
+```
+
+{{% /tab-content %}}
+
+<!-------------------------------- BEGIN Mailgun ---------------------------->
+
+{{% tab-content %}}
+
+The example below uses the Mailgun API to send an alert email when more than 3 critical statuses occur within 10 minutes.
+
+{{% note %}} To view your Mailgun API keys, sign in to Mailjet and open [Account Security - API security](https://app.mailgun.com/app/account/security/api_keys). <br> 
+Mailgun requires that a domain be specified via Mailgun. A domain is automatically created for you when you first set up your account. You must include this domain in your url endpoint (for example, `https://api.mailgun.net/v3/YOUR_DOMAIN` or `https://api.eu.mailgun.net/v3/YOUR_DOMAIN`. If you're using a free version of Mailgun, you can set up a maximum of five authorized recipients (to receive email alerts) for your domain. To view your Mailgun domains, sign in to Mailgun and view the [Domains page](https://app.mailgun.com/app/sending/domains). 
+{{% /note %}}
+
+```js
+import "http"
+
+// Import the Secrets package if you store your API key as a secret.
+// For detail on how to do this, see Step 4 above.
+import "influxdata/influxdb/secrets"
+
+// Retrieve the secret if applicable. Otherwise, skip this line
+// and add the API key as the Bearer token in the Authorization header.
+SENDGRID_APIKEY = secrets.get(key: "SENDGRID_APIKEY")
+
+numberOfCrits = from(bucket: "_monitoring")
+	|> range(start: -10m)
+	|> filter(fn: (r) => (r["_measurement"] == "statuses"))
+	|> filter(fn: (r) => (r["_level"] == "crit"))
+	|> count()
+
+numberOfCrits
+	|> map(fn: (r) =>
+		(if r._value > 1 then {r with _value: http.post(url: "https://api.mailgun.net/v3/YOUR_DOMAIN/messages", headers: {Authorization: "Basic api:your-private-api-key"}, data: bytes(v: "{
+          from='Excited User <mailgun@YOUR_DOMAIN_NAME>' \
+          to=YOU@YOUR_DOMAIN_NAME \
+          to=bar@example.com \
+          subject='Hello' \
+          text='Testing some Mailgun awesomeness!’
+              
```

Comment:
example curl from Mailgun's API:
```curl -s --user ‘api:api-key' \
    https://api.mailgun.net/v3/YOUR_DOMAIN_NAME/messages \
    -F from='Excited User <mailgun@YOUR_DOMAIN_NAME>' \
    -F to=YOU@YOUR_DOMAIN_NAME \
    -F to=bar@example.com \
    -F subject='Hello' \
    -F text='Testing some Mailgun awesomeness!’
```

For API requests, Mailgun says ["No need to craft a MIME (message body) - we assemble it on our side!"](https://help.mailgun.com/hc/en-us/articles/202464990-How-do-I-start-sending-email). 

Would it make sense to include key-value in the url and send with content-type header url-encode? 
Or perhaps drop message body here and refer how to set up messages in Mailgun?

Reply:
It depends on their API. I think we show just a basic example here and then point users to their documentation for more advanced API usage.